src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c

```diff
@@ -139,15 +139,16 @@ void DoorShutter_SetupAction(DoorShutter* this, DoorShutterActionFunc actionFunc
 }
 
 s32 func_808A0900(DoorShutter* this, PlayState* play) {
-    TransitionActorEntry* transitionEntry = &play->doorCtx.transitionActorList[DOORSHUTTER_GET_FC00(&this->actor)];
+    TransitionActorEntry* transitionEntry =
+        &play->doorCtx.transitionActorList[DOOR_GET_TRANSITION_ID(&this->door.dyna.actor)];
```

Comment:
Given the door centric nature of this PR, I'd like to propose a change I implemented in my [z_maps pr](https://github.com/zeldaret/mm/blob/98c44d571c32e58a3a5e09feab9d7796e515dc60/include/z64scene.h#L300) some months back, which is to rename DoorContext -> TransitionActorList and restructure appropriately, simplifying access to e.g. `&play->.transitionActors.List[]`

My argument for this change is that the struct contains only two properties which form a list (the elements ptr and the list count), and it is generally misleading to call all transition actors "doors".

Reply:
This PR is not centered in doors. The door changes are a sideeffect of the changes made to accommodate better how player interacts with doors.
Since player doesn't need any change related to the doorCtx itself, the change you are proposing is out of scope for this PR.

Reply:
`DOORSHUTTER_GET_FC00` is being renamed and relocated, so I figured it'd be an appropriate topic to bring up, as I'm not sure that `DOOR_GET_TRANSITION_ID` needs to be associated with doors.